frontend/src/theme/primitives/typography.ts

```diff
@@ -27,24 +27,26 @@ export const genericFonts = {
   headingFont: fonts.sansSerif,
 }
 
+// Same as in variables.scss
+const fontSizeTwoSmall = 12
+const fontSizeSmall = 14
+
 export const fontSizes = {
-  twoXS: "0.375rem",
-  halfSmDefault: "0.4rem",
-  twoThirdSmDefault: "0.533333rem",
-  xs: "0.5rem",
-  sm: "0.75rem",
-  smDefault: "0.8rem",
+  twoSm: `${fontSizeTwoSmall}px`, // Use px to force sm to be a round number.
+  sm: `${fontSizeSmall}px`, // Use px to force sm to be a round number.
```

Comment:
Just pointing out that this has the same challenges here...perhaps there's a rounder rem number that gives a clear version?

Reply:
Closing this since the above is resolved.